.travis.yml

```diff
@@ -18,7 +18,7 @@ deploy:
     keep_history: true
     local_dir: ./docs/_build/html/
     on:
-      branch: develop
+      branch: master
```

Comment:
This should still be `develop`, yes?

Reply:
I thought we'd want to deploy docs out of master.  It'd be really cool if we could deploy from different locations to different locations, but I think that would be done in a new feature PR.